example/pubspec.yaml

```diff
@@ -35,6 +36,10 @@ dev_dependencies:
     sdk: flutter
   flutter_lints: 1.0.4
 
+dependency_overrides:
+  instabug_flutter:
+    path: ../
+
```

Comment:
The override path matches the dependency path.

Reply:
This is needed because `instabug_http_client` depends on a package version on pub.dev so we need to add the dependency overrides.